### PR TITLE
Update ubuntu setup

### PR DESCRIPTION
**Pull request by murilovarela**
*  Move files to a proper folder;
* Update ubuntu setup;
* Update Android Virtual Devices setup;
* Add AVD helper shell script;

AVD helper commands:
`create` -  Guide user during AVD creation;
`delete` - Help user to delete an AVD;
`devices` - List avaliable devices;
`list` - List existent AVDs;
`packages` - List installed and avaliable Android packages;
`setup` - Help user to setup AVD dependencies;
`start` - Help user to start an existent AVD;

### CODE REVIEW

File: docs/avd [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+    echo "Name: $avd_name"
+    echo "Device: $avd_device"
+    echo "System Image Path: $avd_system_image_path"
+    echo 'Is it correct?[y/n]'
+    read VALID
+
+    if [[ $VALID == 'y' ]]; then
+      echo
+      echo 'Creating Android Virtual Divice...'
```
Comment (willamesoares) [inline docs/avd]: `Device`

File: docs/avd [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+function common_images {
+  echo 'Most common system images:'
+  echo 'system-images;android-21;google_apis;x86'
+  echo 'system-images;android-22;google_apis;x86'
+  echo 'system-images;android-23;google_apis;x86'
+  echo 'system-images;android-24;google_apis_playstore;x86'
+  echo 'system-images;android-26;google_apis_playstore;x86'
+  echo
+  echo 'To list all avaliable system images: use the "packages" command.'
```
Comment (willamesoares) [inline docs/avd]: `available`

File: docs/avd [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+function common_devices {
+  echo 'Most common devices:'
+  echo 'Nexus S'
+  echo 'Nexus 5'
+  echo 'Nexus 5X'
+  echo 'Nexus One'
+  echo 'pixel_xl'
+  echo
+  echo 'To list all avaliable devices: use the "devices" command.'
```
Comment (willamesoares) [inline docs/avd]: `available`

**Comment by vitortalaia** [comment]
@boniattirodrigo Can we merge this one?

File: docs/ubuntu-setup.md [hunk 4]
```diff
@@ -48,20 +48,24 @@ echo "export PATH=$PATH:$ANDROID_HOME/platforms"  >> ~/.bashrc
 ```
 sudo apt-get install android-tools-adb
 ```
-
-## Installation:
+To setup and run an Android Virtual Device see [How To Setup AVD](docs/AVD-setup-ubuntu).
```
Comment (boniattirodrigo) [inline docs/ubuntu-setup.md]: `setup` is a noun, use `set up` because it's a phrasal verb.

**Comment by boniattirodrigo** [comment]
@vitortalaia of course. After resolving this discussion https://github.com/Codeminer42/react-native-starter-kit/pull/12#discussion_r157779540.

**Pull request merged**